cextern/README.rst

```diff
@@ -1,7 +0,0 @@
-External C Libraries
```

Comment:
This feels like an oversight, and probably should be copied back into the template